source/adapter.screenshare.js

```diff
@@ -75,9 +74,9 @@
     };
 
   } else if (window.navigator.webkitGetUserMedia) {
-    baseGetUserMedia = window.navigator.getUserMedia;
 
     navigator.getUserMedia = function (constraints, successCb, failureCb) {
+      baseGetUserMedia = window.navigator.getUserMedia;
```

Comment:
Hum, interesting, in my case, I had to move it in in order to not lose the reference (which I didn;t quite understand at the time)